classes.js

```diff
@@ -0,0 +1,92 @@
+// classes for tic-tac-math
+export {Tile, Board};
+
+
+class Tile {
+    constructor(index) {
+        this.index = index;
+        this.symbol = '';
+        
+    }   
+    
+    tileUnclaimed() {
+        return this.symbol === '';
+    }
+
+    setSymbol(playerSymbol) {
+        if (this.tileUnclaimed()){
+            this.symbol = playerSymbol;
+        }
+        
+    }
+
+    resetSymbol(){
+        this.symbol = '';
+    }
+
+}
+
+class Board {
+    constructor(){
+        this.tiles = [];
+        this.initTiles();       
+    }
+
+    initTiles() {
```

Comment:
can just stick this logic inside the constructor since that's the only place that calls this code.